roborock/typing.py

```diff
@@ -17,192 +17,288 @@
+        prefix=b"\x00\x00\x00\xd7",
+        params=[{"enable": 1, "current_high": 500, "current_integral": 450, "current_low": 400, "stall_time": 10}],
+    ),
+    RoborockCommand.SET_CHILD_LOCK_STATUS: CommandInfo(prefix=b"\x00\x00\x00\x97", params={"lock_status": 0}),
+    RoborockCommand.SET_CLEAN_MOTOR_MODE: CommandInfo(
+        prefix=b"\x00\x00\x00\xb7", params=[{"fan_power": 106, "mop_mode": 302, "water_box_mode": 204}]
+    ),
+    RoborockCommand.SET_COLLISION_AVOID_STATUS: CommandInfo(prefix=b"\x00\x00\x00\x97", params={"status": 1}),
+    RoborockCommand.SET_CUSTOMIZE_CLEAN_MODE: CommandInfo(
+        prefix=b"\x00\x00\x00\xa7", params={"data": [], "need_retry": 1}
+    ),
+    RoborockCommand.SET_CUSTOM_MODE: CommandInfo(prefix=b"\x00\x00\x00\x87", params=[108]),
+    RoborockCommand.SET_DND_TIMER: CommandInfo(prefix=b"\x00\x00\x00\x87", params=[22, 0, 8, 0]),
+    RoborockCommand.SET_DUST_COLLECTION_MODE: CommandInfo(prefix=b"\x00\x00\x00\x87", params=None),
+    RoborockCommand.SET_FDS_ENDPOINT: CommandInfo(prefix=b"\x00\x00\x00\x97", params=["awsusor0.fds.api.xiaomi.com"]),
+    RoborockCommand.SET_FLOW_LED_STATUS: CommandInfo(prefix=b"\x00\x00\x00\x87", params={"status": 1}),
+    RoborockCommand.SET_IDENTIFY_FURNITURE_STATUS: CommandInfo(prefix=b"\x00\x00\x00\x97", params={"status": 1}),
+    RoborockCommand.SET_IDENTIFY_GROUND_MATERIAL_STATUS: CommandInfo(prefix=b"\x00\x00\x00\x97", params={"status": 1}),
+    RoborockCommand.SET_LED_STATUS: CommandInfo(prefix=b"\x00\x00\x00w", params=[1]),
+    RoborockCommand.SET_MOP_MODE: CommandInfo(prefix=b"\x00\x00\x00w", params=None),
+    RoborockCommand.SET_SERVER_TIMER: CommandInfo(prefix=b"\x00\x00\x00\xc7", params=None),
+    RoborockCommand.SET_SMART_WASH_PARAMS: CommandInfo(prefix=b"\x00\x00\x00\x87", params=None),
+    RoborockCommand.SET_TIMEZONE: CommandInfo(prefix=b"\x00\x00\x00\x97", params=["America/Sao_Paulo"]),
+    RoborockCommand.SET_VALLEY_ELECTRICITY_TIMER: CommandInfo(prefix=b"\x00\x00\x00\x97", params=[0, 0, 8, 0]),
+    RoborockCommand.SET_WASH_TOWEL_MODE: CommandInfo(prefix=b"\x00\x00\x00\x87", params=None),
+    RoborockCommand.SET_WATER_BOX_CUSTOM_MODE: CommandInfo(prefix=b"\x00\x00\x00\x87", params=[203]),
+    RoborockCommand.START_CAMERA_PREVIEW: CommandInfo(
+        prefix=b"\x00\x00\x00\xa7", params={"client_id": "443f8636", "quality": "SD"}
+    ),
+    RoborockCommand.START_EDIT_MAP: CommandInfo(prefix=b"\x00\x00\x00w", params=[]),
+    RoborockCommand.START_WASH_THEN_CHARGE: CommandInfo(prefix=b"\x00\x00\x00\x87", params=None),
+    RoborockCommand.STOP_CAMERA_PREVIEW: CommandInfo(prefix=b"\x00\x00\x00\x97", params={"client_id": "443f8636"}),
+    RoborockCommand.SWITCH_WATER_MARK: CommandInfo(prefix=b"\x00\x00\x00\x97", params={"waterMark": "OFF"}),
+    RoborockCommand.TEST_SOUND_VOLUME: CommandInfo(prefix=b"\x00\x00\x00w", params=None),
+    RoborockCommand.UPD_SERVER_TIMER: CommandInfo(prefix=b"\x00\x00\x00\x97", params=None),
 }
```

Comment:
I like this approach  of having the params attached, but I'm confused why they are often pre-filled out? What is the desire with the params attribute?

Also I think it would be helpful to add a description parameter that is defaulted to "", that way we can start describing some of these commands to make it easier during development.